src/global/signal_handler.cc

```diff
@@ -152,6 +158,29 @@ void install_standard_sighandlers(void)
 #include "common/Thread.h"
 #include <errno.h>
 
+void getNameByPid(pid_t pid, char *signal_name) {
+  char proc_pid_path[BUF_SIZE];
+  char buf[BUF_SIZE];
+  sprintf(proc_pid_path, "/proc/%d/status", pid);
```

Comment:
@liewegas Thank you sir! Done it. And  because here modify the signal_handler.cc so the unit test unittest_signals may also need modify. I will do that later.